src/utils/workspace-config/Cargo.toml

```diff
@@ -22,6 +23,7 @@ tracing = { version = "0.1", features = ["release_max_level_debug"] }
 # Static linking
 static-libz-sys = { package = "libz-sys", version = "1", optional = true, features = ["static"] }
 static-lzma-sys = { package = "lzma-sys", version = "0.1", optional = true, features = ["static"] }
+static-openssl-sys = { package = "openssl-sys", version = "0.9.96", optional = true, features = ["vendored"] }
```

Comment:
nit: better to call it “vendored”, because we use env var to make it also static in ci

Reply:
Done: https://github.com/risingwavelabs/risingwave/pull/18689/commits/bc0fc5d8d2f9fb1704e69bc53431db38216d6e57